src/_langs/en/tools/javascript/console/structured-data.markdown

```diff
@@ -15,20 +15,54 @@ priority: 0
 collection: console
 key-takeaways:
   tldr-tbd:
-    - TBD tldr.
+    - Compare similar data objects using `table()`.
```

Comment:
We will want to link this to console API reference docs, and put in complete method name her.